modules/gapi/CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ if(MSVC)
   # Disable obsollete warning C4503 popping up on MSVC <<2017
   # https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4503?view=vs-2019
   ocv_warnings_disable(CMAKE_CXX_FLAGS /wd4503)
+  # FIXME: Unreachable code in Params::backend() (ie.hpp)
+  # https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4702?view=vs-2019
+  ocv_warnings_disable(CMAKE_CXX_FLAGS /wd4702)
```

Comment:
> ie.hpp

Disabling warnings here would not help users when they start building own applications with G-API public headers.
These settings are private for gapi module only.

Reply:
It seems I was able to fix that warning in another header... maybe solution like that would work here too.

Reply:
@TolyaTalamanov Mentioned commit with workaround: https://github.com/opencv/opencv/pull/18584/commits/8637b7ac0b638a474d1a73b571269c11f34c665e